src/components/QuestionList/QuestionList.vue

```diff
@@ -92,7 +93,11 @@ export default {
     },
 
     getUserID() {
-      return localStorage.getItem(this.storageKeyUserID);
+      if(sessionStorage.getItem(this.storageKeyUserID) != null){
+        return sessionStorage.getItem(this.storageKeyUserID);
+      }else{
+        window.alert("Bitte logge dich ein!")
```

Comment:
Please don't use alerts. Check of the user is logged in and then show a different view or redirect somewhere.

getUserID should do what the method is named. Get the user ID.
Nothing else. Please split this code.

Reply:
done?!